modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -5,6 +5,12 @@
 
 This topic lists new features added in Redpanda Cloud.
 
+== January 2024
+
+=== Usage-based billing commitments
+
+You can now sign up for a monthly or an annual usage-based billing commitment to Redpanda Cloud through Google Cloud Marketplace or AWS Marketplace. You can use existing marketplace credits to quickly provision Dedicated Cloud clusters, and you can view your bills and manage your subscription directly in the marketplace.  
```

Comment:
Redpanda Cloud Dedicated now supports usage-based billing. Contact [Redpanda sales](https://redpanda.com/contact) to request a private offer for Monthly and Annual Committed use. Once you receive your private offer from Redpanda, you are able to use existing marketplace credits to quickly provision Dedicated Cloud clusters, and you can view your bills and manage your subscription directly in the marketplace.